src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,54 @@
 import React from 'react';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+// Styles
+import './PostInfo.scss';
+
+// Types
+import { Comment } from '../../Types/comment';
+import { User } from '../../Types/user';
+// Components
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+type Props = {
+  title: string;
+  user: User | null;
```

Comment:
Change 'User | null' to 'User' because user should always be associated with a post.